yarn-project/end-to-end/src/e2e_authwit.test.ts

```diff
@@ -60,12 +60,18 @@ describe('e2e_authwit_tests', () => {
           isValidInPublic: false,
         });
 
+        // We give wallets[0] access to wallets[1]'s notes.
```

Comment:
Might be an absurd amount of work, but I think it would improve readability a lot to do as in other tests and name these wallets (alice, bob...)